apis/apis.json

```diff
@@ -161,8 +161,8 @@
     "id": "Google.Cloud.Logging.Log4Net",
     "version": "1.0.0-beta11",
     "type": "other",
-    "targetFrameworks": "net45",
-    "testTargetFrameworks": "net452",
+    "targetFrameworks": "netstandard1.5;net45"
+    "testTargetFrameworks": "netcoreapp1.0;net452"
```

Comment:
Not that it needs a change now (I don't want to cause more churn than is required :) but these are now the defaults - so we can remove these lines entirely. I'll do that myself in a later commit.